samples/Dantooine/Dantooine.Client/Client/Properties/launchSettings.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
I'm not so sure about this file as VS keeps recreating it if you try to delete it. Is it really unused?

Reply:
yes this is not required, but no idea why it gets created, the server is the only app which is started in the BFF. No idea why this is created or where. I will see if I can find something